lib/build.js

```diff
@@ -443,13 +443,14 @@ function createWebpackMiddleware () {
   const clientCompiler = webpack(clientConfig)
   this.clientCompiler = clientCompiler
   // Add the middleware to the instance context
-  this.webpackDevMiddleware = pify(require('webpack-dev-middleware')(clientCompiler, {
+  this.webpackDevMiddleware = pify(require('webpack-dev-middleware')(clientCompiler, Object.assign(clientConfig.devServer||{},{
```

Comment:
I don't make the rules ;).  All I needed was the 'headers' thing.  I am actually proxying **to** webpack through a separate system, so I needed webpack/nuxt to say "Yeah, that's fine" with CORS headers.